src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.misc.Ui;
```

Comment:
I like the consistency of the the import statements thus far. Perhaps separating the 'import duke.misc.ui' with the 'import javafx...' statements with a new line will make it perfect?

Reply:
Thanks for pointing that out, I totally missed out on that. 😆